### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ghost**
We made our own grid system and with it made simple grid layout.

### CODE REVIEW

**Comment by samgaco** [comment]
### Hi @danilozag1992 and @kobiyoyo :hand: 

### The project looks really good guys, congratulations on that :clap: you have put a lot of work into it. The readme file looks incredible as well :clap:  However, we have to address somethings before the approval:

* The breakpoints should be done taken these measures into consideration. Small screens up to 720px, medium screen sizes 720px to 1024px. Large screen sizes 1024px and above.
* Try to define some more classes on your framework, as some classes for floats, colors (about ten at least), for different borders, hide elements, etc.
* Are you working in shifts? I see only one commit by @danilozag1992 , maybe he could do the next commits for the suggested changes.

### Please :pray: work on these things and submit back for another code review before proceeding.

**Comment by bolah2009** [comment]
Currently under review

File: README.md
Comment (bolah2009) [inline README.md:51]: `AWESOME`

Detailed readme! :+1:

File: css/styles.css
Comment (bolah2009) [inline css/styles.css:78]: `SUGGESTION`

The framework should have been able to do this :thinking: ...

**Review by bolah2009 — APPROVED** [review]
## Nice Job Team :clap: :1st_place_medal: 

The [clone](https://raw.githack.com/kobiyoyo/-GRID-BASED-FRAMEWORK/development/index.html) looks good to me and utilize the CSS Framework nicely. :+1: 

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**
